spec/gutentag/active_record_spec.rb

```diff
@@ -41,6 +42,14 @@
       it { is_expected.not_to include oregon_article }
     end
 
+    context 'given a denormalized tag name' do
```

Comment:
One thing I missed in the previous PR - Melbourne has a U in it, so you'll want to adjust tags/variable names accordingly :)